iwyu.cc

```diff
@@ -714,9 +714,20 @@ class BaseAstVisitor : public RecursiveASTVisitor<Derived> {
       return false;
     if (CanIgnoreCurrentASTNode())
       return true;
-    return this->getDerived().HandleFunctionCall(expr->getDirectCallee(),
-                                                 TypeOfParentIfMethod(expr),
-                                                 expr);
+    FunctionDecl* func = expr->getDirectCallee();
+    if (!func)
+      return true;
+    // If the call makes use of default arguments, select exactly that
```

Comment:
Same as for the new-expr, should be able to make this a simpler loop

Reply:
It looks now really better, thanks!